edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/LocalTwinConfigSource.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core.Config
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+
+    public class LocalTwinConfigSource : IConfigSource
```

Comment:
I wonder if it will be cleaner to make this a TwinConfigSource that always gets the twin from the TwinManager (either cached or the regular call based on a flag), and bypass the EdgeHubConnection altogether.